app/models/placements/mentor.rb

```diff
@@ -21,4 +21,8 @@ class Placements::Mentor < Mentor
   has_many :placements, through: :placement_mentor_joins
 
   default_scope { joins(:mentor_memberships).distinct }
+
```

Comment:
You shouldn't been this.
https://github.com/DFE-Digital/itt-mentor-services/blob/a4022ef4a7c5ab12c3935e8d9e18be846d261fa9/app/models/mentor.rb#L29